### PR TITLE
Adapt to glib::Inhibit removal

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
Depends on https://github.com/gtk-rs/gtk-rs-core/pull/1126

### CODE REVIEW

File: examples/dialog/main.rs [hunk 1]
```diff
@@ -41,7 +40,7 @@ fn build_ui(application: &gtk::Application) {
         if let Some(application) = window.application() {
             application.remove_window(window);
         }
-        Inhibit(false)
+        ControlFlow::Break
```
[the remark this answers is not in the document]
Comment (bilelmoussaoui) [reply examples/dialog/main.rs:43]: Very possibly, I will check tomorrow

[the remark this answers is not in the document]
Comment (andy128k) [reply examples/dialog/main.rs:43]: After looking at related changes I may say the change is correct.

---

But wording seems misleading. "Inhibit" is closer to "Break" (or "Stop" or "Prevent").

The GTK doc of a signal `close-request` says:

> Return value
> Type: gboolean
> TRUE to stop other handlers from being invoked for the signal.

It seems natural to use `Break`  to represent `Inhibit(true)` and `Continue` to represent `Inhibit(false)`.

[the remark this answers is not in the document]
Comment (bilelmoussaoui) [reply examples/dialog/main.rs:43]: Would you mind opening an issue about this?

[the remark this answers is not in the document]
Comment (andy128k) [reply examples/dialog/main.rs:43]: https://github.com/gtk-rs/gtk4-rs/issues/1435

**Pull request merged**